README.md

```diff
@@ -1,3 +1,10 @@
+<p>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/logo.svg">
+    <img alt="Lens SDK logo" src="./assets/logo/green-with-text.svg" width="auto" height="150">
```

Comment:
seems the variant for the light theme is missing.

Reply:
thanks, added a version with a bg